CHANGES.md

```diff
@@ -1,3 +1,39 @@
+## edge-19.9.4
+
+This edge release introduces support for distributed tracing as well as a
+redesigned sidebar in the Web UI!
+
+Support for distributed tracing means that Linkerd data plane proxies can now
+emit trace spans, allowing you to see the exact amount of time spent in the
+Linkerd proxy for traced requests. The new `config.linkerd.io/trace-collector`
+and `config.alpha.linkerd.io/trace-collector-service-account` tracing
+annotations allow specifying which pods should emit trace spans.
+
+The goal of dashboard's sidebar redesign was to reduce load on Prometheus and
+simplify navigation by providing top-level views centered around namespaces and
+workloads.
+
+* CLI
+  * Introduced a new `--cluster-domain` flag to the `linkerd install` command
+    that allows setting a custom cluster domain (thanks @arminbuerkle!)
+  * Added `--disable-heartbeat` flag for `linkerd` `install|upgrade` commands
+* Controller
+  * Instrumented the proxy-injector to provide additional metrics about
+    injection (thanks @Pothulapati!)
+  * Added selectors to MWC that allow skipping namespaces; this fixes `--ha`
```

Comment:
I don't really know what this means.  MWC is mutating webhook controller?  does this just mean the proxy injector?  and how do I add a selector to it?

Reply:
> Added support for `config.linkerd.io/skip-webhook` annotation on namespaces to prevent control plane components from being installed to specific namespaces; this fixes situations in HA deployments where the proxy-injector is installed in `kube-system` (thanks @hasheddan!)